packages/boltz-swaps/tests/submarineRefund.spec.ts

```
@@ -0,0 +1,368 @@
+        privateKey: new Uint8Array(32).fill(1),
+        publicKey: new Uint8Array(33).fill(2),
+    },
+    lockupTxHex: "deadbeef",
+    refundAddress: "bcrt1quser",
+    feePerVbyte: 2,
+    timeoutBlockHeight: 150,
+    ...overrides,
+});
+
+beforeEach(() => {
+    Object.values(mocks).forEach((m) => m.mockReset());
```

Biome (2.5.0)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)